frontend/e2e/tests/project-test.ts

```diff
@@ -20,4 +22,41 @@ export default async function () {
   await click('#save-proj-btn')
   await assertTextContent(`#project-link`, 'Test Project')
 
+  log('Test Change Requests Approvals Setting')
```

Comment:
Sorry, just one little detail here to avoid surprises. We have a feature flag for this feature. Even though it should be enabled, we could only run these tests if it's enabled otherwise they might fail on some environments:
`segment_change_requests`

Reply:
See an example in `/e2e/tests/versioning-tests`

```
    const hasFeature = flagsmith.hasFeature("feature_versioning")
    log('Login')
    await login(E2E_USER, PASSWORD)
    if(!hasFeature) {
        log("Skipping version test, feature not enabled.")
        return
    }
```